packages/manager/modules/iam/src/components/index.js

```diff
@@ -1,11 +1,13 @@
 import iam from './iam';
 import policy from './policy';
 import policies from './policies';
+import policyAdd from './policyAdd';
```

Comment:
I know this is a pain but
Can we rename this to
```suggestion
import createPolicy from './createPolicy';
```

Reply:
It's fixed.